src/main/resources/view/DarkTheme.css

```diff
@@ -282,7 +293,7 @@
 }
 
 .scroll-bar {
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: #9c5b11;
```

Comment:
Great change to the UI! I like the new color scheme. It improves the user's experience!